File: pyg_parent/pyg_interface/src/main/java/com/itheima/pyg/service/seckill/SeckillGoodsService.java
package com.itheima.pyg.service.seckill;

import com.itheima.pyg.pojo.seckill.SeckillGoods;

import java.util.List;

/**
 * 秒杀商品接口
 */
public interface SeckillGoodsService {

    /**
     * 查询所有参与秒杀活动的商品列表
     * @return
     */
    List<SeckillGoods>  findSeckillGoodList();

    List<SeckillGoods>  findSeckillGoodListFromDB();

    /**
     * 根据id查询秒杀商品实体对象
     * @param id
     * @return
     */
    SeckillGoods findOneFromRedis(long id);
}

File: pyg_parent/pyg_web_shop/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function ($http) {

    this.findList = function () {
        return $http.get("../seckillGoods/findList.action");
    };

    this.findOne = function (id) {
        return $http.get("seckillGoods/findOneFromRedis.action?id="+id);
    }
});